### PR TITLE
Implement DimensionTabs component.

### PR DESCRIPTION
**Pull request by kostyalmm**
## Summary

Addresses issue #2424

## Relevant technical choices

* Implement DimensionTabs component.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js [hunk 1]
```diff
@@ -16,6 +16,63 @@
...
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import Tab from '@material/react-tab';
+import TabBar from '@material/react-tab-bar';
```
Comment (eugene-manuilov) [inline assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js:29]: Let's put external dependencies to be above wp dependencies:

```suggestion
/**
 * External dependencies
 */
import Tab from '@material/react-tab';
import TabBar from '@material/react-tab-bar';

/**
 * WordPress dependencies
 */
import { useCallback } from '@wordpress/element';
import { __ } from '@wordpress/i18n';
```

File: assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js [hunk 2]
```diff
@@ -16,6 +16,63 @@
...
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import { FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+const { useSelect, useDispatch } = Data;
+
 export default function DimensionTabs() {
-	return null;
+	const dimensionName = useSelect( ( select ) => select( CORE_FORMS ).getValue( FORM_ALL_TRAFFIC_WIDGET, 'dimensionName' ) || 'ga:channelGrouping' );
```
Comment (eugene-manuilov) [inline assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js:40]: @kostyalmm this is not mentioned in the IB but let's get the `dimensionName` variable from the parent component which also pulls it from the datastore. Let's do it like it's done for `TotalUserCount` and `UserCountGraph` components.

File: assets/sass/widgets/_googlesitekit-widget-analyticsAllTrafficV2.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Analytics AllTraffic V2 styles.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
```
Comment (eugene-manuilov) [inline assets/sass/widgets/_googlesitekit-widget-analyticsAllTrafficV2.scss:4]:
```suggestion
 * Site Kit by Google, Copyright 2020 Google LLC
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @kostyalmm! Mostly looks good to me, just added a few minor comments to address. Could you please check it?

**Review by eugene-manuilov — DISMISSED** [review]
Awesome! Thanks.

**Review by felixarntz — APPROVED** [review]
Awesome!

**Pull request merged**